### DISCUSSION TITLE
Neovim: copilot and macro integration

### DISCUSSION THREAD

**Discussion by coltonmorris** [community/community#7220] [Copilot Conversations]
In neovim, macro's are super powerful. It would be awesome if within a macro we could auto accept copilot suggestions. 

Imagine this macro: Go to next interface in a file, then use repeatable commands to write the comment `DefaultX implements X interface` - then accept the copilot suggestion. 

There are endlessly amazing possibilities here

**Comment by alfans1** [comment]
husseinnooor46@gmail.com
